docs/codeql/ql-language-reference/ql-language-specification.rst

```diff
@@ -7,10 +7,6 @@ QL language specification
 
 This is a formal specification for the QL language. It provides a comprehensive reference for terminology, syntax, and other technical details about QL.
 
-.. This ``highlight`` directive prevents code blocks in this file being highlighted as QL (the default language for this Sphinx project). 
-
-.. highlight:: none
-
```

Comment:
The [global default](https://github.com/github/codeql/pull/4874/files#r547949703) language for this Sphinx project is now "none", so there's no need to repeat this here.